src/Drivers/miniqmc.cpp

```diff
@@ -158,6 +155,7 @@ int main(int argc, char **argv)
       break;
     case 'g': // tiling1 tiling2 tiling3
       sscanf(optarg, "%d %d %d", &na, &nb, &nc);
+      optind += 2;
```

Comment:
The expected form for na,nb,nc separated by spaces pretty much requires them to be enclosed by quotes on the command line (miniqmc -g "2 2 4").  If enclosed in quotes, then they form a single argument.   I don't think incrementing optind is correct here.